Simplified/Book/UI/NYPLBookCellDelegate.m

```diff
@@ -330,12 +331,14 @@ - (void)scheduleTimerForAudiobook:(NYPLBook *)book
   // Target-Selector method required for iOS <10.0
   self.timer = [NSTimer scheduledTimerWithTimeInterval:1.0
                                                 target:self
-                                              selector:@selector(pollAudiobookReadingLocation)
+                                              selector:@selector(pollAudiobookReadingLocation:)
                                               userInfo:nil
                                                repeats:YES];
 }
 
-- (void)pollAudiobookReadingLocation
+// non-thread safe: must be called on the same thread as
+// scheduleTimerForAudiobook:manager:viewController: each time.
+- (void)pollAudiobookReadingLocation:(NSTimer *)timer
```

Comment:
Does adding the `NSTimer` as the parameter do anything here? Just curious

Reply:
no, it does nothing... I merely added it to comply with the docs. But it worked without too.